src/app/app.container.tsx

```diff
@@ -3,13 +3,15 @@ import React, { Component, Fragment } from 'react';
 import { BrowserRouter, Route, Redirect } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { AppState } from '../state';
+import { HomepageContainer } from '../containers/homepage/homepage.container';
 import { CounterContainer } from '../containers/counter/counter.container';
 
 export class AppContainer extends Component {
     render() {
         return (
             <BrowserRouter>
-                <Route path="/" component={CounterContainer} />
+                <Route path="/" component={HomepageContainer} />
```

Comment:
How does routing work here? Will need to read up on that

Reply:
See https://github.com/decrn/localsurvey/pull/24